app/src/main/res/layout/loyalty_card_edit_activity.xml

```diff
@@ -239,7 +245,152 @@
+                </LinearLayout>
+
+                <View
+                    android:id="@+id/divider"
+                    android:layout_width="1dp"
+                    android:layout_height="match_parent"
+                    android:layout_marginTop="3dp"
+                    android:layout_marginStart="8dp"
+                    android:layout_marginEnd="8dp"
+                    android:background="?android:attr/listDivider" />
+
+                <LinearLayout
+                    android:id="@+id/showRemindersLayout"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:paddingHorizontal="@dimen/inputPadding"
+                    android:paddingTop="@dimen/inputPadding"
+                    android:layout_marginTop="5dp"
+                    android:orientation="horizontal">
+
+                    <!-- Show reminders -->
+
+                    <androidx.cardview.widget.CardView
+                        android:id="@+id/reminderCardView"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        app:cardCornerRadius="5dp"
+                        app:cardElevation="1dp"
+                        android:padding="5dp"
+                        android:layout_weight="70"
+                        android:layout_marginBottom="2dp"
+                        android:layout_marginEnd="15dp"
+                        android:visibility="visible">
+
+                        <RelativeLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="match_parent">
+
+                            <TextView
+                                android:id="@+id/daysTextView"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:textColor="@android:color/black"
+                                android:textSize="16sp"
+                                android:layout_marginStart="10dp"
+                                android:layout_marginTop="5dp" />
+
+                            <TextView
+                                android:id="@+id/daysBeforeTextView"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:textSize="16sp"
+                                android:textColor="@android:color/black"
+                                android:layout_marginTop="5dp"
+                                android:paddingStart="4dp"
+                                android:paddingEnd="1dp"
+                                android:paddingBottom="5dp"
+                                android:text="@string/days_before_expiry_date"
+                                android:layout_toEndOf="@id/daysTextView"/>
```

Comment:
daysTextView and daysBeforeTextView should be 1 single field and you should use a formatted plural string like this:

https://github.com/TheLastProject/Catima/blob/11970004f081b401f0157692ee1498d8049f1654/app/src/main/java/protect/card_locker/MainActivity.java#L131

https://github.com/TheLastProject/Catima/blob/5e593d0dede0c1262f763af1aaedf7ef6bf453e8/app/src/main/res/values/strings.xml#L38-L41